Dart/src/com/jetbrains/lang/dart/ide/actions/DartPubActionBase.java

```diff
@@ -147,6 +148,14 @@ public void performPubAction(final @NotNull Module module, final @NotNull Virtua
 
     if (pubParameters != null) {
       final GeneralCommandLine command = new GeneralCommandLine().withWorkDirectory(pubspecYamlFile.getParent().getPath());
+
+      if (FlutterUtil.isFlutterModule(module)) {
+        final String flutterRoot = FlutterUtil.getFlutterRoot(sdk);
+        if (flutterRoot != null) {
+          command.getEnvironment().put("FLUTTER_ROOT", flutterRoot);
```

Comment:
If Flutter be ever supported on Windows it may expect native slashes in env vars. It may be safer to wrap path with `FileUtil.toSystemDependentName(flutterRoot)`.